pipeline.py

```diff
@@ -1518,38 +1518,11 @@ def build_all_images(
         build_image(image, build_configuration)
 
 
-def calculate_images_to_build(
-    images: List[str], include: Optional[List[str]], exclude: Optional[List[str]]
-) -> Set[str]:
-    """
-    Calculates which images to build based on the `images`, `include` and `exclude` sets.
-
-    >>> calculate_images_to_build(["a", "b"], ["a"], ["b"])
-    ... ["a"]
-    """
-
-    if not include and not exclude:
-        return set(images)
-    include = set(include or [])
-    exclude = set(exclude or [])
-    images = set(images or [])
-
-    for image in include.union(exclude):
-        if image not in images:
-            raise ValueError("Image definition {} not found".format(image))
-
-    images_to_build = include.intersection(images)
-    if exclude:
-        images_to_build = images.difference(exclude)
-    return images_to_build
-
-
 def main():
     _setup_tracing()
 
     parser = argparse.ArgumentParser()
-    parser.add_argument("--include", action="append")
-    parser.add_argument("--exclude", action="append")
+    parser.add_argument("--include", required=True)
```

Comment:
Nice! I was planning to do this myself some time ago :D 

What do you think about renaming this argument to `--image`? It will better show that we can only pass single image. `--include` argument is only used in `Makefile` and in https://github.com/mongodb/mongodb-kubernetes/blob/7fa6687dd12bc523f6d5df834672c849b1b737ed/.evergreen-functions.yml#L516

Reply:
I did this already :p
I just did it in small PRs because I was not sure upfront which changes work.
Here is the PR: https://github.com/mongodb/mongodb-kubernetes/pull/132

https://github.com/mongodb/mongodb-kubernetes/pulls/SimonBaeumer